ckanext/switzerland/dcat/processors.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import print_function
+
+import xml
+import logging
+import subprocess
+
+import rdflib
+import rdflib.parser
+
+from ckanext.dcat.utils import url_to_rdflib_format
+from ckanext.dcat.processors import RDFParserException, RDFParser
+
+from shaclprocessor import ShaclParser
+import helpers as tk_dcat
+
+log = logging.getLogger(__name__)
+
+
+class SwissDCATRDFParser(RDFParser):
+    '''
+    An RDF to CKAN parser based on rdflib
+
+    Supports different profiles which are the ones that will generate
+    CKAN dicts from the RDF graph.
+    '''
+
+    def parse(self, data, _format=None,
+              page_count=None,
+              harvest_source_id=None,
+              harvest_job_id=None,
+              shacl_validation=False,
+              shacl_file=None):
+        '''
+        Parses and RDF graph serialization and into the class graph
```

Comment:
I don't understand this sentence

Reply:
This is copied from the original function, therefore I leave it as such, but marked what was copied and what is special. The same applies for the comment below.